packages/graphql-typescript-definitions/test/cli.test.ts

```diff
@@ -27,6 +27,12 @@ describe('cli', () => {
       await execDetails(cliCommandForFixtureDirectory('missing-types')),
     ).toMatchSnapshot();
   });
+
+  it('fails when the file has different name than the query defined within it', async () => {
+    expect(
+      await execDetails(cliCommandForFixtureDirectory('misnamed-file')),
+    ).toMatchSnapshot();
```

Comment:
[We do not use Jest snapshot tests](https://github.com/Shopify/web-foundation/blob/master/Decision%20records/03%20-%20We%20do%20not%20use%20Jest%20snapshot%20tests.md) but we still do. 
IMO, we should try to stop the bleeding from getting worse. Can we refactor this test to not use snapshots. We can revisit the already existing ones later.

Reply:
I'm okay with this if an issue to refactor is raised.